Add EOPF-Zarr GDAL Driver Container with complete environment setup

Yuvraj198920 opened the pull request.

Yuvraj198920 commented:
@rtmiz I've just rechecked, and everything appears to be working now.

I would welcome any insights you might have.

@clausmichele

rtmiz commented:
I'll have to see about including these changes in the default image, but for now would a URL which opens the notebook using the correct image suffice?

Yuvraj198920 commented:
@rtmiz I've added the EOPF-Zarr driver container to our existing container ecosystem:

Integration Approach
Extended CI/CD: Added build-zarr-driver-image job to existing build.yml workflow
Maintained Structure: Preserved original dask and jupyterlab builds while adding eopfzarr-driver
Automatic Triggers: Builds trigger on changes to eopf-zarr-driver/** directory

@clausmichele @konstntokas
